src/Microsoft.Health.Dicom.Api.UnitTests/Controllers/OperationsControllerTests.cs

```diff
@@ -138,35 +141,39 @@ public async Task GivenDoneState_WhenGettingState_ThenReturnOk(OperationStatus d
             Type = DicomOperation.Reindex,
         };
 
-        mediator
+        _mediator
             .Send(
                 Arg.Is<OperationStateRequest>(x => x.OperationId == id),
-                Arg.Is(controller.HttpContext.RequestAborted))
+                Arg.Is(_controller.HttpContext.RequestAborted))
             .Returns(new OperationStateResponse(expected));
 
-        IActionResult response = await controller.GetStateAsync(id);
+        IActionResult response = await _controller.GetStateAsync(id);
         Assert.IsType<ObjectResult>(response);
-        Assert.False(controller.Response.Headers.ContainsKey(HeaderNames.Location));
+        Assert.False(_controller.Response.Headers.ContainsKey(HeaderNames.Location));
 
         var actual = response as ObjectResult;
         Assert.Equal((int)HttpStatusCode.OK, actual.StatusCode);
         Assert.Same(expected, actual.Value);
 
-        await mediator.Received(1).Send(
+        await _mediator.Received(1).Send(
             Arg.Is<OperationStateRequest>(x => x.OperationId == id),
-            Arg.Is(controller.HttpContext.RequestAborted));
-        urlResolver.DidNotReceiveWithAnyArgs().ResolveOperationStatusUri(default);
+            Arg.Is(_controller.HttpContext.RequestAborted));
+        _urlResolver.DidNotReceiveWithAnyArgs().ResolveOperationStatusUri(default);
     }
 
-    [Fact]
-    public async Task GivenSucceededState_WhenGettingState_ThenReturnCompleted()
+    [Theory]
+#pragma warning disable CS0618 // Type or member is obsolete
+    [InlineData(null, OperationStatus.Completed)]
+    [InlineData(1, OperationStatus.Completed)]
+#pragma warning restore CS0618 // Type or member is obsolete
+    [InlineData(2, OperationStatus.Succeeded)]
+    [InlineData(3, OperationStatus.Succeeded)]
+    public async Task GivenSucceededState_WhenGettingState_ThenReturnProperDoneStatus(int? version, OperationStatus expectedStatus)
```

Comment:
The list of operation statuses is here: https://github.com/microsoft/healthcare-shared-components/blob/cd5e98c8fd9434449df8120e5b9b29fc34763add/src/Microsoft.Health.Operations/OperationStatus.cs#L13